Source/EPUBCore/FREpubParser.swift

```diff
@@ -21,15 +21,22 @@ class FREpubParser: NSObject, SSZipArchiveDelegate {
     Returns a FRBook.
     */
     func readEpub(epubPath withEpubPath: String) -> FRBook {
+        
         epubPathToRemove = withEpubPath
         
         // Unzip   
         let bookName = (withEpubPath as NSString).lastPathComponent
-        bookBasePath = (kApplicationDocumentsDirectory as NSString).stringByAppendingPathComponent(bookName)
-        SSZipArchive.unzipFileAtPath(withEpubPath, toDestination: bookBasePath, delegate: self)
+        bookBasePath = (kApplicationDocumentsDirectory as NSString).stringByAppendingPathComponent(bookName+".dir")
         
-        // Skip from backup this folder
-        addSkipBackupAttributeToItemAtURL(NSURL(fileURLWithPath: bookBasePath, isDirectory: true))
+        // unzip the epub if we have not already done so
+        var isDir : ObjCBool = true
+        if NSFileManager.defaultManager().fileExistsAtPath(bookBasePath, isDirectory: &isDir) == false {
```

Comment:
Yes, but on sample is different, unless that we copy the epub to /documents.

This happen became while opening the path is from app, but after remove it moves to /documents. On a real scenario this is not happening, all epub files needs to be at /documents directory. If you put a lot of epubs on app directory I don't think apple will approve it.